Formula/p/python-toml.rb

```diff
@@ -4,29 +4,36 @@ class PythonToml < Formula
   url "https://files.pythonhosted.org/packages/be/ba/1f744cdc819428fc6b5084ec34d9b30660f6f9daaf70eead706e3203ec3c/toml-0.10.2.tar.gz"
   sha256 "b3bda1d108d5dd99f4a20d24d9c348e91c4db7ab1b749200bded2f839ccbe68f"
   license "MIT"
-  head "https://github.com/uiri/toml.git", branch: "master"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "96e51e41b34e32f8b186368deb71ead8c489d8dddf6e0d13bfbef3ef3fd12b99"
   end
 
-  depends_on "python@3.11"
+  depends_on "python@3.10" => [:build, :test]
+  depends_on "python@3.11" => [:build, :test]
+  depends_on "python@3.9" => [:build, :test]
```

Comment:
I believe we wanted to prune out older Python version usage. Previously, we only kept around `python@3.9` usage due to being part of dependencies of disabled formulae, e.g.
- `six` - `ansible@2.9` (removal 2024-05-11) / `q` (removal 2023-12-30)
- `libxml2` - `anjuta` (removal 2023-12-31)
- `pyqt@5` - `vtk@8.2` (removal 2023-11-20)

Once dependent was removed, the plan was to delete `python@3.9` dependency.

Especially with `python@3.12` upcoming, we don't need an excess number of copies of libraries.

Reply:
the main reason of adding more python variants is because of it is the python library.